src/templates/substance.html

```diff
@@ -499,16 +511,30 @@ <h2>Studies & References</h2>
                 }
 
                 // Handle min/max object
-                if (doseValue.min !== undefined && doseValue.max !== undefined) {
-                    return {
-                        low: doseValue.min,
-                        high: doseValue.max,
-                        unit: unit,
-                        display: doseValue.min + '-' + doseValue.max + ' ' + unit
-                    };
+                // if no min/max data then just return null
+                if (doseValue.min == undefined && doseValue.max == undefined) {
+                    return null;
                 }
 
-                return null;
+                /* min OR max can be set seemingly, so need to check each property */
+                // set doseValue.max if min set
+                if (doseValue.min != undefined && doseValue.max == undefined) {
+                    doseValue.max = doseValue.min;
+                }
+
+                // set doseValue.min if max set
+                if (doseValue.min == undefined && doseValue.max != undefined) {
+                    doseValue.min = doseValue.max;
+                }
```

Comment:
Wait, is this how the chart behaved before for unknown values? 

I feel like setting the min value to the max value if only the max value is known is somewhat misleading / potentially dangerous, wouldn't a better solution be to add "unknown" support for the charts themselves to indicate when the min/max isn't known?